template_feed/Microsoft.DotNet.Common.ItemTemplates/content/EditorConfig/.template.config/template.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Config"
+  ],
+  "name": "EditorConfig file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Creates an .editorconfig file for .NET project.",
+  "tags": {
+    "type": "item"
+  },
+  "sources": [
+    {
+      "source": "./Dotnet",
+      "target": "./",
+      "condition": "!Empty"
+    },
+    {
+      "source": "./Empty",
+      "target": "./",
+      "condition": "Empty"
+    }
+  ],
+  "identity": "Microsoft.Standard.QuickStarts.EditorConfigFile",
+  "shortName": "editorconfig",
+  "symbols": {
+    "Empty": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "defaultIfOptionWithoutValue": "true",
+      "displayName": "Empty",
+      "description": "Creates empty .editorconfig instead of .NET specific."
```

Comment:
```suggestion
      "description": "Creates an empty .editorconfig instead of a .NET specific one."
```

Reply:
Text is now updated with very different message